src/main/java/de/tum/in/www1/artemis/web/rest/FileResource.java

```diff
@@ -201,6 +209,24 @@ public FileResource(FileService fileService, ResourceLoader resourceLoader, User
         return responseEntityForFilePath(Constants.DRAG_ITEM_FILEPATH + filename);
     }
 
+    /**
+     * GET /files/file-upload/submission/:submissionId/:filename : Get the file upload exercise submission file
+     *
+     * @param submissionId ID of the submission, the attachment belongs to
+     * @param filename  the filename of the file
+     * @return The requested file, 403 if the logged in user is not allowed to access it, or 404 if the file doesn't exist
+     */
+    @GetMapping("/files/file-upload-submission/{submissionId}/{filename:.+}")
+    @PreAuthorize("permitAll()")
+    public ResponseEntity getFileUploadSubmission(@PathVariable Long submissionId, @PathVariable String filename) {
```

Comment:
This implementation is missing the temporary access token: https://github.com/ls1intum/Artemis/pull/696/files#diff-32ac644447ff79fbf11c9c178d4ab71fR245

Reply:
Good suggestion, added temporary access token